### PR TITLE
Marshal int64 values to JSON strings, treat nb* columns as int32 and i* columns as int32 or float32

### PR DESCRIPTION
**Pull request by zenovich**

### CODE REVIEW

File: app/api/items/add-item.go [hunk 1]
```diff
@@ -14,20 +15,20 @@ import (
...
 		Title       types.RequiredString `json:"title"`
 		ImageURL    types.OptNullString  `json:"image_url"`
 		Subtitle    types.OptNullString  `json:"subtitle"`
 		Description types.OptNullString  `json:"description"`
 	} `json:"strings"`
 
 	Parents []struct {
-		ID    types.RequiredInt64 `json:"id"`
-		Order types.RequiredInt64 `json:"order"`
+		ID    types.RequiredString `json:"id"`
```
[the remark this answers is not in the document]
Comment (zenovich) [reply app/api/items/add-item.go]: fixed

File: app/api/items/get-item.go [hunk 2]
```diff
@@ -92,32 +92,32 @@ func (srv *Service) getItem(rw http.ResponseWriter, httpReq *http.Request) servi
 	}
 
 	user := srv.GetUser(httpReq)
-	userDefaultLanguageID, err := user.DefaultLanguageID()
+	_, err := user.DefaultLanguageID() // check that the user exists
```
[the remark this answers is not in the document]
Comment (zenovich) [reply app/api/items/get-item.go]: > If you want to just check that it exists, create an appropriate method for this.

fixed

[the remark this answers is not in the document]
Comment (smadbe) [reply app/api/items/get-item.go]: And can you answer the second comment?

[the remark this answers is not in the document]
Comment (zenovich) [reply app/api/items/get-item.go]: I have no idea why we load users lazily. Probably, there was a reason... I can try to get rid of User.lazyLoadData() if you will.

[the remark this answers is not in the document]
Comment (smadbe) [reply app/api/items/get-item.go]: Will be done in a separate PR

**Pull request merged**